### DISCUSSION TITLE
ControlNet Edit button is not responding.

### DISCUSSION THREAD

**Discussion by AlpDindar** [Mikubill/sd-webui-controlnet#3007] [Q&A]
I am a novice user, the edit button that worked yesterday does not work today. I uninstalled and reinstalled the plugin but it doesn't work.

I would be grateful if you can help me.

**Comment by CeroxeAnivie** [comment]
我解决了这个问题！替换为最新的1.10.0-RC没有这个毛病！！！
![dc5cd37742d905c1c11c57ef169af53](https://github.com/user-attachments/assets/9ffded68-3c4f-4cb2-88d0-fe472ded7120)

**Reply by AlpDindar** [reply]
I tried that, but it didn't work.
